lib/engine-util.js

```diff
@@ -0,0 +1,34 @@
+// Block colors
+const R = 'red';
```

Comment:
The colors should be customizable in the engine. Not in the scope of this PR though.